average_temp_cdo.R

```diff
@@ -0,0 +1,103 @@
+library(ncdf4)
+library(ggplot2)
+library(dplyr)
+
+#get_weighted_area:
+#Calculates and saves the fraction of each grid that is land and ocean in decimal form (max 1, min 0) and saves both
+#inputs:
+#     land_frac: .nc file location that contains fraction of grid that is land (either in decimal or percent)
+#     land_area: .nc file location that will contain the area of each gris square weighted by the percent of the grid that is land
+#     ocean_frac: .nc file location that will contain the area of each gris square weighted by the percent of the grid that is
+#     cleanup: a boolean value - if true than intermediate files will be deleted - defaults to true
+#outputs:
+#     land_area.nc and ocean_area.nc: as specified above - saved at path_name
+
+get_weighted_areas = function(land_frac, land_area, ocean_area, cleanup = TRUE){
```

Comment:
Use `<-` instead of `=` for the function assignment, but `=` (as you did) for the parameter default assignment. It's not just R style, there's a good (albeit subtle) reason for this.